.github/workflows/test.yml

```diff
@@ -129,10 +133,40 @@ jobs:
          - enabled: "false"
            name: "noHA"
         gateway-mode: [local, shared]
+        ipfamily:
+         - ip: ipv4
+           name: "IPv4"
+           ipv4: true
+           ipv6: false
+         - ip: ipv6
+           name: "IPv6"
+           ipv4: false
+           ipv6: true
+         - ip: dualstack
+           name: "Dualstack"
+           ipv4: true
+           ipv6: true
+        exclude:
+         - {"ipfamily": {"ip": dualstack}}
+         - {"ipfamily": {"ip": ipv6}, "target": {"shard": shard-ipv4}}
+         - {"ipfamily": {"ip": ipv4}, "ha": {"enabled": "true"}, "gateway-mode": shared}
+         - {"ipfamily": {"ip": ipv4}, "ha": {"enabled": "false"}, "gateway-mode": local}
```

Comment:
missing trailing `}`

Reply:
No, these are fine. They are just different than the others because the "gateway-mode" has a value instead of a sub-variable.

Reply:
my fault, didn't realize of the difference